src/libs/actions/Inbox.js

```diff
@@ -29,7 +27,7 @@ function requestInboxCall({
         .then((result) => {
             if (result.jsonCode === 200) {
                 Growl.success(Localize.translateLocal('requestCallPage.growlMessageOnSave'));
-                Report.fetchOrCreateChatReport([email, CONST.EMAIL.CONCIERGE], true);
```

Comment:
Were we only calling `fetchOrCreateChatReport()` here to navigate to the Concierge chat? Or did we also need to sometimes create it?

Reply:
I don't see why we'd need to create it in this scenario anymore. We'll take you back to whatever page you came from after the call is requested. We aren't navigating you to your Concierge chat here, so I think we're good.